cookbooks/aws-parallelcluster-slurm/libraries/helpers.rb

```diff
@@ -178,6 +178,11 @@ def wait_cluster_ready
   end
 end
 
+def get_static_node_count
+  cmd = Mixlib::ShellOut.new("#{cookbook_virtualenv_path}/bin/python #{node['cluster']['scripts_dir']}/slurm/pcluster_fleet_config_generator.py  --input-file #{node['cluster']['cluster_config_path']} --get-total-min-count | grep -o 'The total MinCount of cluster is \([0-9]*\)' | cut -d' ' -f7")
```

Comment:
It seems to me we are introducing too much complexity to determine the total min count. What about parsing the config file directly in YAML?